aria-practices.html

```diff
@@ -4235,6 +4235,16 @@ <h4><span id="naming_role_guidance_heading">Accessible Name Guidance by Role</sp
                 </ul>
               </td>
             </tr>
+            <tr>
+              <td><a href="#blockquote" class="role-reference"><code>blockquote</code></a></td>
+              <td>Discretionary</td>
+              <td>
+                <ul>
+                  <li>Naming is optional, but can potentially help screen reader users understand the context and purpose of the quote.</li>
+                  <li>Named using <code>aria-labelledby</code> if a visible label is present, otherwise with <code>aria-label</code>.</li>
```

Comment:
Discourage aria-label: discretionary is kinda positive. This wording may encourage people to label all their blockquotes. Doesn't improve the experience.

Reply:
"If a visible label is present, associating it with the blockquote by using aria-labelledby could benefit some AT users."